### PR TITLE
Star Light

### PR DESCRIPTION
**Pull request by Azrun**
## About the PR 
image

## Why's this needed? 
Improves visibility of anything adjacent to fullbright space.  Should provide some situational awareness in areas that have been spaced.

Adjusting the robustlight to the fluid color seemed to improve quality of coloration but seemed quite minor.

### CODE REVIEW

File: code/turf/turf.dm [hunk 1]
```diff
@@ -276,6 +281,21 @@ proc/generate_space_color()
...
 
+/turf/space/proc/update_starlight(starlight_alpha=255)
+	color = space_color
+	if(fullbright)
+		if(!starlight)
+			starlight = image('icons/effects/overlays/simplelight.dmi', "3x3", pixel_x=-32, pixel_y=-32)
+			starlight.appearance_flags = RESET_COLOR | RESET_TRANSFORM | RESET_ALPHA | NO_CLIENT_COLOR | KEEP_APART
+			starlight.layer = LIGHTING_LAYER_BASE
+			starlight.plane = PLANE_LIGHTING
+			starlight.blend_mode = BLEND_ADD
```
Comment (pali6) [inline code/turf/turf.dm:292]: Originally I thought you'd just straight up use `src.add_simple_light`, though I guess it makes sense to keep this as slim as possible considering how much space there is. Then I thought this could be simplified by making it so `starlight` is actually of type `/obj/overlay/simple_light` instead of `/image` and while I think this would work properly it's also a bit confusing since UpdateOverlays expects its argument to be an image (but I don't see any obvious place where anything would fail if it were an atom). So *then* I thought you could just keep it an image but do `var/obj/overlay/simple_light/simple_light/light_dummy = new` and `starlight = light_dummy.appearance` but annoyingly enough if we are staying true to type safety then appearance "objects" are not actually /image either despite being very similar in many respects.

TLDR: you probably don't need to touch these lines at all, they are ok.

**Review by pali6 — APPROVED** [review]

**Comment by pali6** [comment]
Is UpdateIcon called in some parent New?

File: code/turf/turf.dm (hunk 1 above)
Comment (Azrun) [reply code/turf/turf.dm:289]: I believe all the existing lightning systems assume color will be provided as R, G, B and not hex/matrix/etc.  I actually tried converting to simple lights and then was playing with the color transform and 💥.
